.cirrus.yml

```diff
@@ -24,3 +24,16 @@ task:
   test_script:
     - . $HOME/.cargo/env
     - sh ci/run.sh x86_64-unknown-freebsd
+
+task:
+  name: nightly i686-unknown-freebsd
+  setup_script:
+    - pkg install -y curl
+    - curl https://sh.rustup.rs -sSf --output rustup.sh
+    - sh rustup.sh -y
+    - . $HOME/.cargo/env
+    - rustup default nightly
+    - rustup target add i686-unknown-freebsd
+  test_script:
+    - . $HOME/.cargo/env
+    - sh ci/run.sh i686-unknown-freebsd
```

Comment:
I suspect that this will fail.  Because while FreeBSD amd64 can run i386 binaries, it doesn't include i386 headers, and libc's CI tests need to interrogate those headers.  Testing FreeBSD i386 in CI will require an actual i386 environment, either a VM or a jail on an amd64 host.

Reply:
Ideally we would use `qemu-user` for this, we need something like that to test `aarch64-unknown-freebsd` and friends as well.

Reply:
qemu isn't necessary for running i386 binaries on an amd64 host; that capability is builtin.  All you need is an i386 jail, or even a chroot.  See https://github.com/nix-rust/nix/blob/master/.cirrus.yml for an example of testing i386 binaries for a project that doesn't need headers.

Reply:
We do need headers, so that `.cirrus.yml` does not help much.